src/q3/methodnum-FSAB1104/summary/methodnum-FSAB1104-summary.tex

```diff
@@ -502,30 +509,31 @@ \subsection{Extrapolation de Richardson}
 
 
 	\end{tikzpicture}
-	\label{Richardson}
-	\caption{Extrapolation de Richardson}
+	\caption{Extrapolation de Richardson.}
+     \label{fig:richardson}
 \end{figure}
 
 % TODO : ajouter l'interprétation graphique de l'extrapolation de Richardson
 
 \subsection{Méthode de Romberg}
 Soit $f(h) = I^h$, l'intégrale exacte est $f(0)$.
 On peut donc utiliser Richardson en calculant
-$I^h$ avec la méthode des trapèze.
+$I^h$ avec la méthode des trapèzes.
 C'est ce qu'on appelle la méthode de Romberg.
 
 Cependant, $E^h$ n'a pas de terme d'ordre impair, on peut donc passer
 une étape sur deux dans Richardson.
-On utilise donc la méthode adaptée suivante pour la récurrence
+On utilise donc la méthode adaptée suivante pour la récurrence:
 \[ I_{i, j} = \frac{2^{2j} I_{i, j-1} - I_{i-1, j-1}}{2^{2j} - 1} \]
-$I - I_{n, n}$ est alors de grand ordre $\bigoh\left(h^{2(n+1)}\right)$.
+$I - I_{n, n}$ est alors de grand ordre $\bigoh\left(h^{2(n+1)}\right).$.
 
 \part{Dérivation numérique}
 Taylor nous permet, avec un peu d'astuce, d'obtenir des
 expressions pour les dérivées de $u$.
 Sans perte de généralité, on va les calculer autour de 0.
 
 \subsubsection{Différences centrées}
```

Comment:
"Difference (3) between bookmark levels is greater than one, level fixed".
==> Rajoute un `\section{Différences}` et remplace les `\subsubsection` par des `\subsection`.

Reply:
(C'est le seul qui justifie le "request change"